component/tests/common.py

```diff
@@ -83,28 +82,6 @@ def setUp(self):
         )
 
 
-class SavepointComponentCase(common.SavepointCase, ComponentMixin):
```

Comment:
@yankinmax this is still needed but should inherit from `common.TransactionCase` and be renamed to `ComponentCase` or `TransactionComponentCase`.
Is not the same one as `TransactionComponentRegistryCase`.

Reply:
I left TransactionComponentCase and removed SavepointComponentCase. Just open code of commit pls.

Reply:
I'm looking at the full diff. There's no class named `TransactionComponentCase` and should stay here.
As said, this is not the same as `TransactionComponentRegistryCase`.

Reply:
-https://github.com/yankinmax/connector/blob/370efd265e3aa5637aa5918da3b6fb9ce99cd913/component/tests/common.py#L59

Reply:
ah yeah, sorry, the expanding of the diff tricked me. That class is already good as is.